hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/VirtualdatasourceConfig.java

```diff
@@ -16,8 +16,10 @@
 
 package com.hedera.node.config.data;
 
+import com.hedera.node.config.NodeProperty;
 import com.swirlds.config.api.ConfigData;
 import com.swirlds.config.api.ConfigProperty;
 
 @ConfigData("virtualdatasource")
-public record VirtualdatasourceConfig(@ConfigProperty(defaultValue = "true") boolean jasperdbToMerkledb) {}
+public record VirtualdatasourceConfig(
+        @ConfigProperty(defaultValue = "true") @NodeProperty boolean jasperdbToMerkledb) {}
```

Comment:
I believe converting between DB types would be a network-wide change.

Reply:
Probably so!

Reply:
Updated.